x/sysproxy/doc.go

```diff
@@ -0,0 +1,39 @@
+/*
+Package sysproxy provides a simple interface to set/unset system-wide proxy settings.
+
+# Platform Support
+Currently this package supports desktop platforms only. The following platforms are supported:
+- macOS
+- Linux (Gnome is the only supported desktop environment for now.)
+- Windows
+
+# Usage
+
+- To set up system-wide proxy settings, use the `SetProxy` function. This function takes two arguments: the IP address and the port of the proxy server.
+`SetProxy(ip string, port string) error`
```

Comment:
Use `[...]` to create an in-page link to the exported function, so the document reader can simplify click it and navigate to the corresponding function signature.

```suggestion
- To set up system-wide proxy settings, use the [SetProxy] function. This function takes two arguments: the IP address and the port of the proxy server.
```